app/views/admin/environments/_fields.html.erb

```diff
@@ -0,0 +1,20 @@
+<fieldset>
+  <div class="form-group">
+    <%= form.label :name, "Name", class: "col-lg-2 control-label" %>
+    <div class="col-lg-4">
+      <%= form.text_field :name, class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <div class="col-lg-offset-2 col-lg-2">
+      <div class="checkbox">
+        <%= form.label :is_production do %>
```

Comment:
this guy should be a static display once you added a deploy group, so you can still see it but no longer modify flip it